TABTestKit/Classes/Contexts/NavigationContext.swift

```diff
@@ -58,43 +58,43 @@ public extension NavigationContext {
-    /// or a fresh launch of the app.
-    ///
-    /// Since you can pass any URL to this function, you could pass an app-specific url like `myapp://deep-link`, or
-    /// a regular HTTP link like `https://apple.com`.
-    ///
-    /// HTTP links are opened in Safari by default, unless you have an app installed that can intercept them using
-    /// Universal Links.
-    ///
-    /// - Parameter url: The URL to open.
-    func open(_ url: URL) {
-        springboard.activate()
-        if !Icon.testRunner.determine(.hittable, timeout: 1) {
-            XCUIDevice.shared.press(.home) // Ensure we're on the first page of the home screen
-            sleep(1)
-            while !Icon.testRunner.determine(.hittable, timeout: 1) {
-                springboard.swipeLeft()
-            }
-        }
-        Icon.testRunner.tap()
-        sleep(1)
-        UIApplication.shared.open(url)
-        let openAppConfirmationAlert = Alert(id: "“\(Bundle.main.appName)” wants to open “\(App.shared.name)”", parent: springboard)
-        if openAppConfirmationAlert.determine(.exists, timeout: 1) {
-            sleep(1)
-            openAppConfirmationAlert.actionButton(withID: "Open").tap()
-        }
-    }
-    
+	}
+	
+	/// Opens the specified URL from the test runner app.
+	/// This means you can use this function for testing deep linking, as well as opening web urls in Safari.
+	/// This function will first show the Home Screen if necessary, launch the test runner (after finding the correct
+	/// page with the runner) and then open the URL.
+	///
+	/// It is your responsibility/choice if you want to leave your app in the current state before the test runner
+	/// opens the URL, or whether you want to completely terminate your app before hand.
+	///
+	/// This leaves you free to test deep linking behaviour from the current state of the app.
+	/// or a fresh launch of the app.
+	///
+	/// Since you can pass any URL to this function, you could pass an app-specific url like `myapp://deep-link`, or
+	/// a regular HTTP link like `https://apple.com`.
+	///
+	/// HTTP links are opened in Safari by default, unless you have an app installed that can intercept them using
+	/// Universal Links.
+	///
+	/// - Parameter url: The URL to open.
+	func open(_ url: URL) {
+		springboard.activate()
+		if !Icon.testRunner.determine(.hittable, timeout: 1) {
+			XCUIDevice.shared.press(.home) // Ensure we're on the first page of the home screen
+			sleep(1)
+			while !Icon.testRunner.determine(.hittable, timeout: 1) {
+				springboard.swipeLeft()
+			}
+		}
+		Icon.testRunner.tap()
+		sleep(1)
+		UIApplication.shared.open(url)
+		let openAppConfirmationAlert = Alert(id: "“\(Bundle.main.appName)” wants to open “\(App.shared.name)”", parent: springboard)
+		if openAppConfirmationAlert.determine(.exists, timeout: 1) {
+			sleep(1)
+			openAppConfirmationAlert.actionButton(withID: "Open").tap()
+		}
+	}
+	
```

Comment:
```suggestion
    }
    
    /// Opens the specified URL from the test runner app.
    /// This means you can use this function for testing deep linking, as well as opening web urls in Safari.
    /// This function will first show the Home Screen if necessary, launch the test runner (after finding the correct
    /// page with the runner) and then open the URL.
    ///
    /// It is your responsibility/choice if you want to leave your app in the current state before the test runner
    /// opens the URL, or whether you want to completely terminate your app before hand.
    ///
    /// This leaves you free to test deep linking behaviour from the current state of the app.
    /// or a fresh launch of the app.
    ///
    /// Since you can pass any URL to this function, you could pass an app-specific url like `myapp://deep-link`, or
    /// a regular HTTP link like `https://apple.com`.
    ///
    /// HTTP links are opened in Safari by default, unless you have an app installed that can intercept them using
    /// Universal Links.
    ///
    /// - Parameter url: The URL to open.
    func open(_ url: URL) {
        springboard.activate()
        if !Icon.testRunner.determine(.hittable, timeout: 1) {
            XCUIDevice.shared.press(.home) // Ensure we're on the first page of the home screen
            sleep(1)
            while !Icon.testRunner.determine(.hittable, timeout: 1) {
                springboard.swipeLeft()
            }
        }
        Icon.testRunner.tap()
        sleep(1)
        UIApplication.shared.open(url)
        let openAppConfirmationAlert = Alert(id: "“\(Bundle.main.appName)” wants to open “\(App.shared.name)”", parent: springboard)
        if openAppConfirmationAlert.determine(.exists, timeout: 1) {
            sleep(1)
            openAppConfirmationAlert.actionButton(withID: "Open").tap()
        }
    }
    
```